butterknife-compiler/build.gradle

```diff
@@ -13,14 +13,14 @@ for (File file : sdkHandler.sdkLoader.repositories) {
 }
 
 dependencies {
-  compile project(':butterknife-annotations')
-  compile deps.auto.common
-  compile deps.javapoet
+  implementation project(':butterknife-annotations')
+  implementation deps.auto.common
+  api deps.javapoet
```

Comment:
This is implementation too, no? We don't expose these types in the public API.

Reply:
@JakeWharton `implementation deps.javapoet; testRuntimeOnly deps.javapoet` so the consumers are not affected, and it reflects the intent?

Reply:
Sure. Send a PR